### PR TITLE
Style/single views

### PR DESCRIPTION
**Pull request by GraceGardner**
____ Wrote Tests __x__ Implemented ____ Reviewed

## Neccesary checkmarks:

    [] All Tests are Passing in all environments

    [x] The code will run locally

## Type of change

    [x] New feature
    [] Bug Fix
    [] Refactor

## Description of Change:

    description: Added styling for single sighting and single cryptid page. Added Error modal and styling.
    
## Requested feedback: 
    
    I'd like feedback on... anything that y'all see that may be fishy

## Check the correct boxes

    [x] This broke nothing
    [] This broke some stuff
    [] This broke everything

## Testing Changes

    [x] No Tests have been changed
    [] Some Tests have been changed
    [] All of the Tests have been changed(Please describe what in the world happened)

## Checklist:

    [] My code has no unused/commented out code
    [x] My code has no binding.pry's
    [x] I have reviewed my code
    [] I have commented my code, particularly in hard-to-understand areas
    [] I have fully tested my code

(For Fun!)Please Include a link to a gif of your feelings about this branch

Link:https://media.giphy.com/media/90oi2KtSymXtu/giphy.gif

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -4,19 +4,26 @@ import { Route, Routes } from 'react-router-dom';
...
   return (
-    <NavigationProvider>
-      <NavBar />
-      <Routes>
-        <Route path="/" element={<Home />} />
-        <Route path="/singlecryptid/:name" element={<SingleCryptid />} />
-        <Route path="/allsightings" element={<AllSightings />} />
-      </Routes>
-    </NavigationProvider>
+    <ErrorProvider>
```
Comment (krdurrett) [inline src/App.js:15]: Ohhhh thanks for adding the error provider context api! Also the single sighting component!

File: src/Components/ErrorModal.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    setError()
+  }
+
+  return (
+    <>
+    {error &&
+      <div className='error-modal'>
+        <div className='modal-content'>
+          <p className='error-message'>{error}</p>
+            <button className='error-button'
```
Comment (krdurrett) [inline src/Components/ErrorModal.js:19]: Beautiful error component!

File: src/Components/TrailCard.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+      </div>
+      <div className='trail-details-container'>
+        <p className='trail-name'>fake card Text</p>
+        <p className='trail-link'>fake link</p>
+      </div>
+    </div>
+  )
+}
+
+export default SightingCard;
```
Comment (krdurrett) [inline src/Components/TrailCard.js:19]: Should this be TrailCard rather than SightingCard?

File: src/Styles/_mixins.scss [hunk 4]
```diff
@@ -14,6 +14,22 @@
...
+  box-shadow: 0px 14px 29px -1px rgba(0,0,0,0.65);
+  border: none;
+  padding: 19%;
+}
+
+@mixin button-hover {
+  cursor: pointer;
+  opacity: 0.7;
+}
+
```
Comment (krdurrett) [inline src/Styles/_mixins.scss:32]: Thank you for doing all the styling!

**Review by krdurrett — APPROVED** [review]
Just a bit confused about the TrailCard vs the SightingCard?

File: src/Components/ErrorModal.js [hunk 5]
```diff
@@ -0,0 +1,30 @@
+import React, {useContext} from 'react';
+import { Link } from 'react-router-dom';
+import {ErrorContext} from '../Context/ErrorContext'
+import '../Styles/ErrorModal.scss';
+
+const ErrorModal = () => {
+  const {error, setError} = useContext(ErrorContext)
+
+  const clearError = () => {
```
Comment (jasmyn2244) [inline src/Components/ErrorModal.js:9]: This is very cool! Can't wait to see the error modal in action.

File: src/Components/TrailCard.js (hunk 3 above)
Comment (jasmyn2244) [reply src/Components/TrailCard.js:19]: Good catch!

File: src/Styles/Card.scss [hunk 6]
```diff
@@ -3,12 +3,33 @@
 
 .card {
   @include gold-gradient;
+  border-radius: 5px;
```
Comment (jasmyn2244) [inline src/Styles/Card.scss:6]: So excited to see the styling!

**Review by jasmyn2244 — APPROVED** [review]
I can't wait to see all of this in action!

**Pull request merged**